apps/s_server.c

```diff
@@ -147,13 +147,14 @@ static int dtlslisten = 0;
 static char *psk_identity = "Client_identity";
 char *psk_key = NULL;           /* by default PSK is not used */
 
-static unsigned int psk_server_cb(SSL *ssl, const char *identity,
+static unsigned int psk_server_cb(SSL *s, const char *identity,
```

Comment:
You are being inconsistent with what you said/implied before :-) You said that your preferences are with argument *names* in declarations and definitions matching exactly. And there is support for this in style guide. Well, specifically here there is a level of indirection, i.e. it is the pointer to this function that is used, but I'd expect you to express your preferences even here :-) As already mentioned earlier, myself I'd argue that argument name matching is not in contradiction with style guide in cases of indirection like above, so I'd accept it. Though I'd say that if renamed `ssl_unused` would be a 100 times better name in this case. But that's just me.

BTW, *if* style guide is amended to permit unused pre-/suffix, wouldn't it be more optimal to harmonize with it here at once? I mean as opposite to committing this now and then going through it once again later on?